package.json

```diff
@@ -1,67 +1,84 @@
 {
-  "name": "link-preview-js",
-  "version": "3.0.5",
-  "description": "Javascript module to extract and fetch HTTP link information from blocks of text.",
-  "main": "build/index.js",
+  "name": "@dscvr-one/link-preview-js",
+  "description": "JavaScript module to extract and fetch HTTP link information from blocks of text.",
+  "version": "0.0.0",
+  "sideEffects": false,
```

Comment:
what does `sideEffects` do?

Reply:
Basically tells we aren’t doing any global imports or css imports to cause side effects while tree shaking

good explanation: https://stackoverflow.com/a/49776306